### DISCUSSION TITLE
Power this board from a 5V USB Cable and get rid of the battery?

### DISCUSSION THREAD

**Discussion by gramss** [alvarop/usb_c_cable_tester#5] [Ideas]
Hi,

just a simple question as I would like to order your board, but just want to get rid of the battery situation.

What do you think about it? :)

Cheers

**Comment by alvarop** [comment]
Totally doable! Should be pretty easy for you to copy/paste one of the USB-C connectors and connect VBUS from it instead of the battery :D
If you clone the git repo it should just open with KiCad 6
